openedx/features/course_experience/views/course_reviews.py

```diff
@@ -45,10 +46,12 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
 
         """
         course_key = CourseKey.from_string(course_id)
-        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
+        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=False)
         course_url_name = default_course_url_name(course.id)
         course_url = reverse(course_url_name, kwargs={'course_id': unicode(course.id)})
 
+        is_enrolled = CourseEnrollment.is_enrolled(request.user, course.id)
```

Comment:
This doesn't seem to be used.  Is it needed?  Was the idea not show for students that weren't enrolled?  If so, should you just set `course_reviews_provider_fragment` to None?

Also, do we need a test to verify whatever this should be doing?